backend/src/services/GenerateReporter/GenerateReportService.ts

```diff
@@ -299,7 +299,7 @@ export class GenerateReportService {
       kanbanSetting.boardColumns,
       kanbanSetting.users
     );
-    this.jiraColumns = await kanban.getJiraColumns(
+    this.jiraColumns = await kanban.getColumns(
```

Comment:
could you please rename the `jiraColumns` also? if it is not displayed only for Jira

Reply:
renamed jiraColumns to columns and changed type JiraColumnResponse to ColumnResponse as well.